applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/multiprotocol_rpmsg.conf

```diff
@@ -0,0 +1,30 @@
+# Configuration increases maximum data length of PDU supported in the Controller
+# and reduces number of Bluetooth connections to limit memory usage.
+
+## Disable serial and UART interface.
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+
+CONFIG_LOG=n
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=2
+CONFIG_BT_CTLR_ASSERT_HANDLER=y
+CONFIG_BT_HCI_RAW_RESERVE=1
+
+# Increase maximum data length of PDU supported in the Controller
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+
+CONFIG_ASSERT=y
```

Comment:
Asserts increase the size of binary and decrease the performance. This is a copy/paste from the original. Maybe @Damian-Nordic and guys would like to disable such things? I recommend to do it in a separate PR as it is not related.